.github/ISSUE_TEMPLATE/adicionar_evento .yml

```
@@ -0,0 +1,182 @@
+    validations:
+      required: false
+
+  - type: markdown
+    attributes:
+      value: |
+        ## Quando o evento vai acontecer?
+
+  - type: dropdown
+    id: event_year
+    attributes:
+      label: "Ano do Evento"
+      description: "Qual o ano que irá acontecer o evento?"
+      multiple: false
+      default: 0
+      options:
+        - "2025"
+    validations:
+      required: true
+
+  - type: dropdown
+    id: event_month
+    attributes:
+      label: "Mês do Evento"
+      description: Em qual mês irá acontecer o evento?
+      multiple: false
+      default: 0
+      options:
+        - A anunciar
+        - Janeiro
+        - Fevereiro
+        - Março
+        - Abril
+        - Maio
+        - Junho
+        - Julho
+        - Agosto
+        - Setembro
+        - Outubro
+        - Novembro
+        - Dezembro
+    validations:
+      required: true
+
+  - type: dropdown
+    id: event_day
+    attributes:
+      label: "Dia do Evento"
+      description: Selecione os dias em que o evento irá ocorrer. (Múltipla escolha)
+      multiple: true
+      options:
+        - "01"
+        - "02"
+        - "03"
+        - "04"
+        - "05"
+        - "06"
+        - "07"
+        - "08"
+        - "09"
+        - "10"
+        - "11"
+        - "12"
+        - "13"
+        - "14"
+        - "15"
+        - "16"
+        - "17"
+        - "18"
+        - "19"
+        - "20"
+        - "21"
+        - "22"
+        - "23"
+        - "24"
+        - "25"
+        - "26"
+        - "27"
+        - "28"
+        - "29"
+        - "30"
+        - "31"
+    validations:
+      required: true
+
+  - type: markdown
+    attributes:
+      value: |
+        ## Horário do Evento
+
+  - type: input
+    id: event_start_time
+    attributes:
+      label: "Horário de Início"
+      description: "Qual horário o evento irá começar?"
+      placeholder: "ex: 14:00"
+    validations:
+      required: true
+
+  - type: input
+    id: event_end_time
+    attributes:
+      label: "Horário de Término"
+      description: "Qual horário o evento irá terminar?"
+      placeholder: "ex: 18:00"
+    validations:
+      required: true
+
+  - type: markdown
+    attributes:
+      value: |
+        ## Detalhes Adicionais
+
+  - type: textarea
+    id: event_description
+    attributes:
+      label: "Descrição do Evento"
+      description: "Conte mais sobre o evento."
+      placeholder: "ex: Este evento é sobre..."
+    validations:
+      required: true
+
+  - type: input
+    id: event_organizer
+    attributes:
+      label: "Nome do Organizador"
+      description: "Quem está organizando o evento?"
+      placeholder: "ex: João Silva"
+    validations:
+      required: true
+
+  - type: input
+    id: event_organizer_contact
+    attributes:
+      label: "Contato do Organizador"
+      description: "Como podemos entrar em contato com o organizador?"
+      placeholder: "ex: joao@email.com"
+    validations:
+      required: true
```

Ainda ficou com o aviso 😅 haha Se você só aceitar a minha sugestão aqui, vai dar boa:

```suggestion
      required: true

```